static/js/config/post-processing.js

```diff
@@ -14,9 +14,9 @@ MEDUSA.config.postProcessing = function() { // eslint-disable-line max-lines
     })();
 
     function isRarSupported() {
-        $.get('config/postProcessing/isRarSupported', function(data) {
-            if (data !== 'supported') {
-                $('#unpack').qtip('option', {
+        $.post('config/postProcessing/isRarSupported', {unrar_tool: $('#unrar_tool').val()}, function(data) { // eslint-disable-line camelcase
```

Comment:
Here `config/postProcessing/isRarSupported` is a route. The same as `config/postProcessing/savePostProcessing` and `home/testSABnzbd` these are all in camelCase. So this is correct, or at least following the current convention.